Image processing/Image compression analyser/tests/test_compressors.py

```
@@ -0,0 +1,304 @@
+import unittest
+import tempfile
+import os
+import sys
+from PIL import Image
```

Ruff (0.12.2)

`PIL.Image` imported but unused

Remove unused import: `PIL.Image`

(F401)